src/components/Toast/src/Toast.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+	<m-bread
+		:class="[
+			$s.Toast,
+			{
+				[$s.EnhanceContrast]: enhanceContrast,
+			},
+		]"
+		:persistent="resolvedPersistent"
+		:dismiss-after="resolvedDismissAfter"
+		:style="style"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div :class="$s.ToastContent">
+			<m-icon
```

Comment:
Can we make icon a prop as well as a slot? Similar to what we did with the Accordion header. So it's not necessary to add an icon to a theme if it's a toast has a one-off icon?

Reply:
hmmm, i kinda would prefer people to just use the `iconName` slot buuut since we have them as slots in Accordion and Notice i'll add it into Toast for consistency's sake